orca_detector/orca_utils.py

```diff
@@ -0,0 +1,102 @@
+"""
+Utility functions for the Orca project.
+
+W251 (Summer 2019) - Spyros Garyfallos, Ram Iyer, Mike Winton
```

Comment:
If this is your code from another class, this should be reflected here

Reply:
done